substrate/frame/support/src/migrations.rs

```diff
@@ -777,8 +777,10 @@ impl<T: SteppedMigration> SteppedMigrations for T {
 		1
 	}
 
-	fn nth_id(_n: u32) -> Option<Vec<u8>> {
-		Some(T::id().encode())
```

Comment:
This was the actual issue

Reply:
Oh and this was not overwritten for a single migration that's not part of a tuple, right?